README.md

```diff
@@ -143,21 +176,27 @@ You must have [Docker installed](https://docs.docker.com/engine/installation/) t
     git clone git@github.com:quipucords/quipucords-ui.git
     ```
 
-2. *Optional* - Build UI:
-    ```
+2. Build UI:
+   
+   On Mac: 
+   ```
     brew install yarn (if you don't already have yarn)
     make build-ui
     ```
+   On Linux:
+   ```
+   dnf install yarn (if you don't already have yarn)
```

Comment:
This works but since we are recommending to use asdf to manage node I think we should point the user to use that installation to get yarn :thinking: 

```
asdf shell nodejs 14.18.3
npm install -g yarn
```

Reply:
Updated document to use asdf instead.